### PR TITLE
remove v8 update and refactored code

### PR DESCRIPTION
**Pull request by vairafiq**
## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Security fix
- [ ] Improvement
- [ ] New Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Text changes
- [ ] Other... Please describe:

## Description
How to reproduce the issue or how to test the changes

1.
2.
3.

## Any linked issues
Fixes #

## Checklist

- [ ] My code follows the [WordPress coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)

### CODE REVIEW

**Comment by Armanul46** [comment]
Works fine

File: includes/classes/class-installation.php [hunk 1]
```diff
@@ -45,9 +45,6 @@ class ATBDP_Installation {
 			'directorist_7123_remove_upload_files_cap',
 			'directorist_7123_update_db_version',
 		],
-		'8.0.0' => [
```
Comment (obiPlabon) [inline includes/classes/class-installation.php]: Restore the db version update

File: includes/classes/class-upgrade.php [hunk 2]
```diff
@@ -30,6 +31,253 @@ public function __construct()
...
+
+	public function v8_force_migration() {
+
+		if( get_option( 'directorist_v8_force_migrated' ) ) {
+			return;
+		}
+
+		$listings = get_posts( 
+			[
+				'post_type' => 'at_biz_dir',
```
Comment (obiPlabon) [inline includes/classes/class-upgrade.php]: Disable pagination query and caching.

**Review by obiPlabon — CHANGES_REQUESTED** [review]
Please update the changes

File: includes/classes/class-upgrade.php (hunk 2 above)
Comment (obiPlabon) [reply includes/classes/class-upgrade.php]: Please take a look on the following query params to understand in details
`nopaging`, `cache_results`, `update_post_meta_cache`, `update_post_term_cache`

**Review by obiPlabon — APPROVED** [review]

**Pull request merged**